flutter/pubspec.yaml

```diff
@@ -41,3 +41,5 @@ flutter:
       web:
         pluginClass: SentryFlutterWeb
         fileName: sentry_flutter_web.dart
+      linux:
+        pluginClass: SentryFlutterPlugin
```

Comment:
should we maybe name all the plugins a bit more specific to the platform? like SentryFlutterLinux or, to rename the others to be always generic? eg SentryFlutterPlugin

Reply:
If these names are just for internal use (I believe that's the case) it would be beneficial to align on the side of appending the platform so make it easier to navigate through the project

Reply:
Yep, these are internal names. They however don't necessarily map as is to a class or file name. On linux there is for example a struct named after this, but the file is named `sentry_flutter_plugin.cc` and `sentry_flutter_plugin.h`.
Also MacOS and iOS use the same file.

We can align these name but I don't really think it matters that much, because the files are also in their corresponding platform folders. Your call :D

Reply:
its just a matter of consistency, not really "necessary", I agree with @bruno-garcia that it helps to navigate thru the project, even more, when you take care of more repos, which does the same thing, with similar names